gym-unity/gym_unity/envs/__init__.py

```diff
@@ -371,13 +371,13 @@ def _sanitize_info(self, step_result: BatchedStepResult) -> BatchedStepResult:
                 "The number of agents in the scene does not match the expected number."
             )
 
-        if step_result.n_agents() - sum(step_result.done) != self._n_agents:
+        if step_result.n_agents() - sum(step_result.status) != self._n_agents:
```

Comment:
This is going to sum the enum values, right? I don't think that's what you want.

Reply:
It is not. This is definitely a bug here.

Reply:
🔥